src/app/credExplorer/App.js

```diff
@@ -62,6 +63,18 @@ export default class App extends React.Component<Props, State> {
           ) : (
             <p>Graph not loaded.</p>
           )}
+          <button
+            disabled={graph == null}
+            onClick={() => {
+              setTimeout(() => {
+                if (graph != null) {
+                  basicPagerank(graph);
```

Comment:
This should set the stationary distribution in the page state.

Reply:
See my comment on https://github.com/sourcecred/sourcecred/pull/252#discussion_r187194747.